jvm/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -356,6 +356,11 @@ object ConverterUtils extends Logging {
 
   // SparkSQL Math fucctions of Velox
   final val ABS = "abs"
+  final val CEIL = "ceil"
+  final val FLOOR = "floor"
+  final val EXP = "exp"
+  final val POWER = "power"
+  final val PMOD = "pmod"
```

Comment:
Have you checked these functions in Substrait yaml?

Reply:
Only pmod() need to add in Substrait yaml, will create PR to Substrait.